### PR TITLE
Insert Futurice Logo

### PR DESCRIPTION
**Pull request by miro**
screen shot 2015-03-18 at 15 47 14

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -52,7 +52,21 @@ let view = Cycle.createView(model => {
...
   function renderHeader(progress) {
-    return h('.row', [
+    return [
+    h('.row', [
+      h('.col-md-12', [
+        h('h2', {
+          style: mergeStyles(textStyle, {
+            'height': '45px',
+            'padding': '12px 0 0 212px',
+            'background-image': 'url(futu-logo.png)',
```
Comment (staltz) [inline src/app.js]: You can write `backgroundImage` instead of `'background-image'` ;) @miro

Comment (staltz) [inline src/app.js]: Also, `dist/img/futu-logo.png` ?

**Comment by staltz** [comment]
I think I'll move the logo and "Helsinki moves" to the top left corner of the body, to avoid too much stuff near the progress bar.

File: src/app.js (hunk 1 above)
Comment (miro) [inline src/app.js]: Both changed, thanks for the feedback

**Comment by staltz** [comment]
How do you think we should combine this PR and #2 ?

**Comment by miro** [comment]
@staltz, I think the logo could be moved to the "empty space above the progress bar":

screen shot 2015-03-18 at 16 41 48

And maybe make it somehow transparent / otherwise glitter-ify it. And make it bigger.

**Comment by staltz** [comment]
How about use the circular futu logo on the top left corner ? The favicon of www.futurice.com is what I'm talking about

**Pull request closed without merging**